camel/messages/base.py

```diff
@@ -116,19 +126,6 @@ def __contains__(self, item: str) -> bool:
         """
         return item in self.content
 
-    def token_len(self, model: ModelType = ModelType.GPT_3_5_TURBO) -> int:
```

Comment:
Why remove this?

Reply:
It was unused in the code. Do you think, we'd better keep it?

Reply:
What about adding it back right after it is needed at least once?

Reply:
I didn't see the value of removing it. It is useful and we will for sure use it later. So let's keep it.

Reply:
Ok. I bring it back